src/scanner.cc

```diff
@@ -1092,8 +1092,19 @@ Parser pop =
 /**
  * Advance the lexer until the following character is neither space nor tab.
  */
-Parser skipspace =
-  effect([](State & state) { while (cond::peek(' ')(state) || cond::peek('\t')(state)) state::skip(state); });
+void skipspace(State &state) {
+  while (true) {
+    switch (PEEK) {
+      case ' ':
+      case '\t':
+        S_ADVANCE;
+        break;
+      default:
+        return;
```

Comment:
I'd use `break` here for consistency. (Or `return` in both situations.)

Reply:
Hmm, this returns from the function, the previous break breaks only out of the switch (and continues the `for` loop).

Reply:
Ah yes, overlooked that!